views/default/default/getAdvice.ejs

```diff
@@ -119,7 +119,7 @@
       <strong>UTS Library's</strong> <a href="http://www.lib.uts.edu.au/about-us/staff-profiles/information-services/research-data-team" target="_blank"> Research Data Team </a>is responsible for providing <a href="http://www.lib.uts.edu.au/research/data-management/" target="_blank">training and consultation on RDM</a> across the research project lifecycle.
       </li><li><strong>UTS eResearch</strong> are dedicated to helping researchers with research-specific tools, technology and training; a little like IML for researchers.
       eResearch provides researchers with a number of <a href="<%= BrandingService.getBrandAndPortalPath(req) %>/availableServicesList"> research platform services </a>
-      which meet the requirements of the <a href="http://www.gsu.uts.edu.au/policies/research-management-policy.html" target="_blank"> UTS Research Management Policy (2018)</a> and accompanying <a href="https://staff.uts.edu.au/topichub/RIO%20Library/RIO%20documents/Research%20environment/New%20research%20directives/Research%20Management%20Guidelines/Guidelines%20for%20the%20Management%20of%20Research%20Data%20v0%2084%2020171120.pdf?Web=1" target="_blank">Guidelines for Research Data Management (opens in StaffConnect)</a>.
+      which meet the requirements of the <a href="https://gsu.uts.edu.au/policies/research.html" target="_blank"> UTS Research Management Policy (2021)</a> and accompanying <a href="https://staff.uts.edu.au/topichub/RIO%20Library/RIO%20documents/Research%20environment/New%20research%20directives/Research%20Management%20Guidelines/Guidelines%20for%20the%20Management%20of%20Research%20Data%20v0%2084%2020171120.pdf?Web=1" target="_blank">Guidelines for Research Data Management (opens in StaffConnect)</a>.
```

Comment:
Looks like that StaffConnect link is dead, probably replaceable by referencing the mainly "Research Policy" and the RDM procedures
```suggestion
      which meet the requirements of the <a href="https://gsu.uts.edu.au/policies/research.html" target="_blank"> UTS Research Policy (2021)</a> and accompanying <a href="https://gsu.uts.edu.au/policies/documents/research-data-management-procedures.pdf" target="_blank">Research Data Management Procedures</a>.
```